backend/src/v5/models/modelSettings.js

```diff
@@ -72,4 +72,37 @@ Models.getFederations = async (ts, ids, projection, sort) => {
 	return findModel(ts, query, projection, sort);
 };
 
+Models.updateModelSettings = async (ts, model, payload) => {
+	const toUpdate = {};
+	const toUnset = {};
+	const keys = Object.keys(payload);
+
+	for (let i = 0; i < keys.length; i++) {
```

Comment:
Probably a bit cleaner to just do a foreach here
```suggestion
	Object.keys(payload).forEach((key) => {
	    //...
	});
```

then you don't need the next line.